src/sage/combinat/posets/hasse_cython_flint.pyx

```diff
@@ -20,6 +20,53 @@ from sage.libs.flint.fmpz_mat cimport *
 from sage.matrix.matrix_integer_dense cimport Matrix_integer_dense
 from sage.matrix.matrix_space import MatrixSpace
 from sage.rings.integer_ring import ZZ
+from sage.libs.flint.fmpz_poly_sage cimport Fmpz_poly
+
+
+cpdef Fmpz_poly chain_poly(list positions):
+    r"""
+    Compute the chain polynomial of a poset.
```

Comment:
Return the chain polynomial of a poset. ?

Reply:
voila